app/screens/PlaceholderScreen.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from "react"
+import { observer } from "mobx-react-lite"
+import { ViewStyle } from "react-native"
+import { StackScreenProps } from "@react-navigation/stack"
+import { AppStackParamList } from "../navigators"
+import { Button, Screen } from "../components"
+import { useNavigation } from "@react-navigation/native"
+import { NativeStackNavigationProp } from "@react-navigation/native-stack"
+import { TabParamList } from "../navigators/TabNavigator"
+// import { useNavigation } from "@react-navigation/native"
+// import { useStores } from "../models"
+
+// STOP! READ ME FIRST!
+// To fix the TS error below, you'll need to add the following things in your navigation config:
+// - Add `Placeholder: undefined` to AppStackParamList
+// - Import your screen, and add it to the stack:
+//     `<Stack.Screen name="Placeholder" component={PlaceholderScreen} />`
+// Hint: Look for the 🔥!
+
+// REMOVE ME! ⬇️ This TS ignore will not be necessary after you've added the correct navigator param type
+// @ts-ignore
+export const PlaceholderScreen: FC<StackScreenProps<AppStackParamList, "Placeholder">> = observer(
+  function PlaceholderScreen() {
+    // Pull in one of our MST stores
+    // const { someStore, anotherStore } = useStores()
+
+    // Pull in navigation via hook
+    const navigation = useNavigation<NativeStackNavigationProp<TabParamList>>()
```

Comment:
Can we add a small custom hook for this to avoid having to add the types every time? Like `useAppNavigation` or somethign.